pkg/translate/field_mask_test.go

```diff
@@ -0,0 +1,282 @@
+package translate
+
+import (
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/exp/maps"
+)
+
+func mergeSlices[T ~[]E, E any](a, b T) T {
+	merged := make(T, len(a)+len(b))
+	copy(merged, a)
+	copy(merged[len(a):], b)
+
+	return merged
+}
+
+func mergeMaps[T ~map[K]V, K comparable, V any](a, b T) T {
```

Comment:
This function uses maps.Copy to merge two maps. It was created for convenience and its name clearly states its purpose. To avoid modifying the original maps a or b, the function creates a new variable. This is because maps.Copy(a,b) appends the keys/values of b to a

Reply:
but not relevant anymore